user/tests/wiring/adc_dac/adc_dac.cpp

```diff
@@ -71,3 +72,20 @@ test(ADC_AnalogReadOnPinWithDACOutputResultsInCorrectValue) {
     //To Do : Add test for remaining pins if required
 }
 #endif
+
+#if (PLATFORM_ID == 6)
```

Comment:
The for loop is to reproduce the failing sequence of operations in issue #671.   In that failure, the DAC would be stuck at a random value between 0 and Vdd, so would fail the test.

The resistor divider setup only applies to Core, which does not have a DAC.  In that case, we do a resistor divider to bias the ADC to 1/2VDD.  100K might be too much in comparison to the input impedance of the ADC, changing it to 5 or 10k should be better.  However, since the test check for a ADC in the range 2000 to 2100, the test should still pass even with the loading of input impedance on A5. 

For the Photon, there is no resistor divider.  The output buffer of DAC drives the ADC directly.